### ISSUE TITLE
skip on import

### ISSUE THREAD

**Issue by zoellner** [skratchdot/elasticsearch-tools#6]
I am running into the issue of getting timeouts after a while (tens of thousands of operations successful, running for hours, then a timeout).
I'd like to restart the bulk import where it was left of before with an option to skip the first n entries. I think that should be an easy addition

**Comment by skratchdot** [comment]
I agree.  Might also be useful to have a transform or filter on import.  Skip seems important though...

For skip to work, probably only need a new skip variable, and some if/else logic around these lines:
https://github.com/skratchdot/elasticsearch-tools/blob/a65fab8a346bbaed1d69cbb7f0a49e1876bf6022/lib/es-import-bulk.js#L88-L89
